docs/src/logging.md

```diff
@@ -1,43 +1,50 @@
 # Logging
 
-The operator is designed to log in JSON format directly to standard output,
-including PostgreSQL logs.
+The operator outputs logs in JSON format directly to standard output, including
+PostgreSQL logs, without persisting them to storage for security reasons. This
+design facilitates seamless integration with most Kubernetes-compatible log
+management tools, including command line ones like
+[stern](https://github.com/stern/stern).
 
-Each log entry has the following fields:
+!!! Important
+    Long-term storage and management of logs are outside the scope of the
+    operator and should be handled at the Kubernetes infrastructure level.
+    For more information, see the
+    [Kubernetes Logging Architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/)
+    documentation.
 
-- `level` – Log level (`info`, `notice`, ...).
+Each log entry includes the following fields:
+
+- `level` – The log level (e.g., `info`, `notice`).
 - `ts` – The timestamp (epoch with microseconds).
-- `logger` – The type of the record (for example, `postgres` or `pg_controldata`).
-- `msg` – The actual message or the keyword `record` in case the message is parsed in JSON format.
-- `record` – The actual record with structure that varies depending on the
+- `logger` – The type of log (e.g., `postgres`, `pg_controldata`).
+- `msg` – The log message, or the keyword `record` if the message is in JSON
+  format.
+- `record` – The actual record, with a structure that varies depending on the
   `logger` type.
-- `logging_podName` – The pod where the log was created.
-
-!!! Warning
-    Long-term storage and management of logs is outside the operator's purview,
-    and needs to be provided at the level of the Kubernetes installation.
-    See the
-    [Kubernetes Logging Architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/)
-    documentation.
+- `logging_pod` – The name of the pod where the log was generated.
 
 !!! Info
-    If your log ingestion system requires it, you can rename the `level` and `ts` field names using the `log-field-level` and
-    `log-field-timestamp` flags of the operator controller. Edit the `Deployment` definition of the
-    `cloudnative-pg` operator.
+    If your log ingestion system requires custom field names, you can rename
+    the `level` and `ts` fields using the `log-field-level` and
+    `log-field-timestamp` flags in the operator controller. This can be configured
+    by editing the `Deployment` definition of the `cloudnative-pg` operator.
 
-## Operator log
+## Operator Logs
```

Comment:
Reading the code, and indeed, the Operator logLevel is not easily configurable. It is not supported in the ConfigMap, and the only real way to configure it is to `Kubectl edit` the deployment and specify the level in the `--log_level` invocation.
In the CI/CD we do configure the level by directly messing with the `--log-level` flag.